src/tasks/MonoTargetsTasks/EmitBundleTask/EmitBundleObjectFiles.cs

```diff
@@ -42,4 +40,8 @@ public override bool Emit(string destinationFile, Action<Stream> inputProvider)
         return exitCode == 0;
     }
 
+    public override string GetDestinationFileExtension()
```

Comment:
Add `label: Path.GetFileName(destinationFile)` in `TryRunProcess` call above, so when running for multiple files the output can be made sense of in the logs.

Reply:
Done!